.github/actions/free-up-space-for-dotnet/FreeUpSpaceUbuntu.ps1

```diff
@@ -0,0 +1,42 @@
+$initialSpace = (df / |
+        Select-Object -Skip 1 |
+        ForEach-Object { $PSItem.Split(' ', [System.StringSplitOptions]::RemoveEmptyEntries)[3] } |
+        Measure-Object -Sum).Sum
+
+df --human-readable
+
+# Remove the local cache of downloaded packages but don't remove installed software.
+sudo apt-get clean
+
+# Remove Android SDK.
+sudo rm -rf /usr/local/lib/android
```

Comment:
Is there a particular reason for using `rm -rf` instead of `Remove-Item -Recurse -Force`? If so, please include a comment.